cmd/calidum-rotae-service/client/client.go

```diff
@@ -0,0 +1,56 @@
+package client
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/clubcedille/calidum-rotae-backend/cmd/calidum-rotae-service/config"
+	discord_provider "github.com/clubcedille/calidum-rotae-backend/pkg/proto-gen/discord-provider"
+	email_provider "github.com/clubcedille/calidum-rotae-backend/pkg/proto-gen/email-provider"
+	"github.com/spf13/viper"
+	"google.golang.org/grpc"
+)
+
+func InitFromViper(ctx context.Context, v *viper.Viper) (
+	discord_provider.DiscordProviderClient,
+	email_provider.EmailProviderClient,
+	error,
```

Comment:
Les autres fonctions du package ont des valeurs de retour nommée, alors on pourrait nommer ceux là aussi juste pour que ça reste cohérent
```suggestion
	discordClient discord_provider.DiscordProviderClient,
	emailClient email_provider.EmailProviderClient,
	err error,
```